src/main/java/pro/sky/Examiner/exception/BadAmount.java

```diff
@@ -0,0 +1,12 @@
+package pro.sky.Examiner.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.client.HttpStatusCodeException;
+
+public class BadAmount extends HttpStatusCodeException {
```

Comment:
```suggestion
public class BadAmountException extends HttpStatusCodeException {
```
т.к. классы-исключения лучше оканчивать на Exception

Reply:
подправила